topics/assembly/tutorials/vgp_workflow_training/tutorial.md

```diff
@@ -183,20 +183,33 @@ The three {HiFi} fasta files are better represented as a collection: {collection
 
 {% snippet faqs/galaxy/collections_build_list.md %}
 
-
 Once we have imported the datasets, the next step is to import the VGP workflows from the WorkflowHub.
 
 # Importing workflows
 
+All analyses described in this tutorial are performed using *workflows*--chains of tools. Before we can proceed we need to import workflows into your Galaxy account. In order to do this you need to follow the instruction below.
```

Comment:
I think there can be better ways to present this, but if "learning to use DS/WFH" is one of your learning objectives for users of this tutorial, then I guess we keep it as is.

E.g. we could be using the same method the GTN uses to link directly to a workflow and have it be 1-click-to-run like we do for workflows inside the GTN, but adapted for WFH/DS.

but we can leave this as-is for now, since that would require some more work